### PR TITLE
DOC: mstats: add examples to kurtosis

### PR DESCRIPTION
**Pull request by daniel-finnan**
#### Reference issue
Towards #7168

#### What does this implement/fix?
Add examples to mstats kurtosis

#### Additional information
[docs only]
First PR, tried to follow contributing guidelines, but apologies in advance if I've messed up!

### CODE REVIEW

File: scipy/stats/_mstats_basic.py [hunk 1]
```diff
@@ -2897,6 +2897,32 @@ def kurtosis(a, axis=0, fisher=True, bias=True):
     -----
     For more details about `kurtosis`, see `scipy.stats.kurtosis`.
 
+    Examples
+    --------
+    In this example, we generate an array using the `scipy.stats.norm.rvs`
```
Comment (tylerjereddy) [inline scipy/stats/_mstats_basic.py:2902]: For the doc build failure you can see in the CI testing, I did a quick check of our previous usages of this:

`git grep -E -i "scipy.stats.norm.rvs"`

```
scipy/stats/_continuous_distns.py:    >>> data = scipy.stats.norm.rvs(size=100000, loc=0, scale=1.5,
scipy/stats/_hypotests.py:    Suppose we wish to test whether data generated by ``scipy.stats.norm.rvs``
scipy/stats/_hypotests.py:    ``scipy.stats.norm.rvs`` have the same distribution. We choose a
scipy/stats/_mstats_basic.py:    In this example, we generate an array using the `scipy.stats.norm.rvs`
```

so, it looks like an extra set of backticks may do the trick!

**Comment by daniel-finnan** [comment]
Thanks for the pointers Tyler, think this should work now

File: scipy/stats/_mstats_basic.py [hunk 2]
```diff
@@ -2897,6 +2897,32 @@ def kurtosis(a, axis=0, fisher=True, bias=True):
...
+    The result is close to zero since we're using Fisher's definition, which
+    uses a zero value as a reference point to the normal distribution, however,
+    this has been impacted by removal of the masked values from the calculation.
+
+    >>> import numpy as np
+    >>> from scipy.stats import norm
+    >>> from scipy.stats.mstats import kurtosis
+    >>> data = norm.rvs(size=10, random_state=5)
+    >>> masked = np.ma.MaskedArray(data, mask=[False, True, False, False, False,
+    >>>                                        False, False, False, True, False])
```
Comment (yuanx749) [inline scipy/stats/_mstats_basic.py:2914]:
```suggestion
    >>> masked = np.ma.MaskedArray(data, mask=[False, True, False, False, False,
    ...                                        False, False, False, True, False])
```

**Review by yuanx749 — CHANGES_REQUESTED** [review]

**Comment by mdhaber** [comment]
Thanks @daniel-finnan, but given the responses so far in gh-22194, I don't think we'll be maintaining entirely separate examples for `mstats` functions and their `stats` counterparts. I'll adjust gh-7168 accordingly.

**Pull request closed without merging**